presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -195,6 +195,9 @@ Property Name                                      Description
                                                    S3SelectPushdown.
 
 ``hive.metastore.load-balancing-enabled``          Enable load balancing between multiple Metastore instances
+
+``hive.skip-empty-files``                          Enable skipping empty files. Otherwise, it will produce an   ``false``
+                                                   error iterating through these files.
```

Comment:
```suggestion
                                                   error iterating through empty files.
```
nit of a suggestion: be specific when possible, especially if it doesn't add length or complexity to do so.